tests/manage/pv_services/test_pvc_resize.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework.testlib import skipif_ocs_version
+
+log = logging.getLogger(__name__)
+
+
+@skipif_ocs_version(['>4.1', '<4.5'])
+@pytest.mark.parametrize(
+    argnames=["interface"],
+    argvalues=[
+        pytest.param(
+            constants.CEPHBLOCKPOOL, marks=pytest.mark.polarion_id("OCS-00000")
+        ),
+        pytest.param(
+            constants.CEPHFILESYSTEM, marks=pytest.mark.polarion_id("OCS-0000")
+        )
+    ]
+)
+def test_pvc_resize(interface, storageclass_factory, pvc_factory, pod_factory):
+    """
+    Verify PVC resize feature
+    """
+    pvc_size = 10
+    pvc_size_new = 40
+    patch_param = f'{{"spec": {{"resources": {{"requests": {{"storage": "{pvc_size_new}Gi"}}}}}}}}'
+
+    sc_obj = storageclass_factory(interface=interface)
+
+    # Create PVC and pod
+    log.info("Creating PVC and pod.")
+    pvc_obj = pvc_factory(interface=interface, storageclass=sc_obj, size=pvc_size)
+    pod_obj = pod_factory(interface=interface, pvc=pvc_obj)
+
+    # Modify size of PVC
+    assert pvc_obj.ocp.patch(resource_name=pvc_obj.name, params=patch_param), (
+        f"Patch command to modify size of PVC {pvc_obj.name} has failed."
+    )
+
+    # Wait for 60 seconds to reflect the change on pod
+    for df_out in TimeoutSampler(
+        60, 3, pod_obj.exec_cmd_on_pod, command='df -kh'
+    ):
+        df_out = df_out.split()
+        new_size_mount = df_out[df_out.index(pod_obj.get_storage_path()) - 4]
+        if new_size_mount == f'{pvc_size_new}G':
+            break
+        log.info(
+            f"New size on mount is not {pvc_size_new}G as expected, "
+            f"but {new_size_mount}. Retrying."
+        )
```

Comment:
It will be great if this functionality can be added too to the ocs_ci.ocs.resources.pvc.PVC#resize_pvc when verify=True

Reply:
Done.